Wake On Lan/Modules/AboutScreen/Tests/AboutScreenTests/AboutScreenPresenterTests.swift

```diff
@@ -0,0 +1,83 @@
+import XCTest
+
+@testable import AboutScreen
+@testable import WOLResources
+
+final class AboutScreenPresenterTests: XCTestCase {
+    var sut: AboutScreenPresenter!
+    var interactorMock: AboutScreenInteractorInputMock!
+    var viewMock: AboutScreenViewInputMock!
+
+    override func setUp() {
+        super.setUp()
+        interactorMock = AboutScreenInteractorInputMock()
+        viewMock = AboutScreenViewInputMock()
+        sut = AboutScreenPresenter()
+        sut.interactor = interactorMock
+        sut.view = viewMock
+    }
+
+    func testViewDidLoad() {
+        // when
+        sut.viewDidLoad(viewMock)
+        // then
+        XCTAssertEqual(interactorMock.fetchBundleInfoCallsCount, 1, "Interactor must be called once")
+    }
+
+    func testIntercatorDidFetchBundleInfo() {
+        // given
+        let bundleInfo = TestData.bundleInfo
+        let viewModel = TestData.viewModel
+        // when
+        sut.interactor(interactorMock, didFetchBundleInfo: bundleInfo)
+        // then
+        XCTAssertEqual(
+            viewMock.configureWithCallsCount,
+            1,
+            "ViewController must be called once"
+        )
+        XCTAssertEqual(
+            viewMock.configureWithReceivedViewModel?.buttonListViewModel.count,
+            viewModel.buttonListViewModel.count,
+            "the number of button items must equal 3"
+        )
+        XCTAssertEqual(
+            viewMock.configureWithReceivedViewModel?.headerViewModel,
+            viewModel.headerViewModel,
+            "Presenter must call ViewController with viewModel made from bundleInfo "
+        )
+    }
+}
+
+private extension AboutScreenPresenterTests {
+    enum TestData {
+        static let bundleInfo = BundleInfo(
```

Comment:
Let's move it to separate file `BundleInfo+Seeds.swift`. Due to this approach won't create the same test data in each test class, instead we will be able to reuse it.

Reply:
Also, I think we should use random strings (i.m.  `displayName: String.random()`) instead of hardcode them. It will make our tests more confident.

Reply:
I created 2 extensions:
1) String.random( length: Int) - to create a random string of a certain length. 
2) BundleInfo + Seeds extension - static let testData which can be used for testing. 
I would appreciate your review on the changes. 
Thank you.